msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1434,7 +1428,7 @@ boolean matches(@NonNull final String username,
     };
 
     protected IAuthenticationResult acquireTokenSilentSync(@NonNull final String[] scopes,
```

Comment:
Not related to this change but why is this `protected` ?

Reply:
This seems to be our internal function (i.e. the public facing interfaces IMulti/ISingle are invoking this, but we don't expose it directly).